src/org/opendatakit/briefcase/ui/automation/AutomationPanel.java

```diff
@@ -0,0 +1,26 @@
+package org.opendatakit.briefcase.ui.automation;
+
+import javax.swing.JPanel;
+import org.bushe.swing.event.annotation.AnnotationProcessor;
+
+public class AutomationPanel {
+  public static final String TAB_NAME = "Automation";
+
+  private final AutomationPanelForm view;
+
+
+  public AutomationPanel(AutomationPanelForm view) {
+    AnnotationProcessor.process(this);
+    this.view = view;
```

Comment:
I was expecting here something like: 
```java
view.onGenerate((AutomationConfiguration config) -> {
  // The stuff from the action listener of the generateScriptButton in AutomationPanelForm
});
```

Reply:
Why can't this be 
```java
view.onGenerate(() -> {

});
```
Like how it's done in the export panel. See response to the question below for an explanation of this.